src/content/en/2021/markup.md

```diff
@@ -2,7 +2,7 @@
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Markup
 description: TODO
-authors: []
+authors: [AlexLakatos]
 reviewers: []
 analysts: []
```

Comment:
@AlexLakatos can you suggest edits to the Authors, Reviewers, and Analysts properties so that they contain everyone who contributed to the chapter, roughly ordered by those with the most significant contributions first? This is what readers will see at the top of the chapter so we want to be sure to give appropriate credit to everyone who helped.

Reply:
Added known authors and analysts. Leaving the reviewers blank for now until the chapter is reviewed.